frontend/app/src/App.tsx

```diff
@@ -154,7 +154,10 @@ interface State {
   hideSidebarNav: boolean
   appPages: IAppPage[]
   navSections: string[]
+  // The hash of the current page executing
   currentPageScriptHash: string
+  // mainScriptHash is the script hash of the main script
```

Comment:
I think we can remove this comment as it does not add any new information in addition to the variable name. what do you think?
I like the comment about

```typescript
// In MPAv2, the main page is executed before the current page
```
much better as it provides some more context information. Perhaps the comment can be extended slightly:

```typescript
// In MPAv2, the main page is executed before the current page. The main page is the script the app is started with, and the current page is the dynamically loaded page-script. In MPAv1, the two hashes should be the same.
```
(only of course if that^ is correct 😄).
Also, the `main page` is executed _before_ and _after_ the current page, if there are directives coming after the `st.Page.run` call?

Reply:
I only modified the ending. "In MPAv1, the main page holds no relevance as only one page is executed at a time."